node-lib/src/runner.rs

```diff
@@ -319,6 +322,27 @@ fn regtest_chain_config(options: &ChainConfigOptions) -> Result<ChainConfig> {
     update_builder!(max_block_size_with_standard_txs);
     update_builder!(max_block_size_with_smart_contracts);
 
+    if chain_pos_netupgrades.unwrap_or(false) {
+        let pos_net_upgrades = vec![
```

Comment:
btw there is `NetUpgrades::unit_test` function. Maybe it's better to create another one `unit_test_with_pos` because I see this code in lots of places

Reply:
Nice.